### ISSUE TITLE
[SUGGESTION] : Diplomacy is lopsided AF

### ISSUE THREAD

**Issue by weirfish** [Stellaris-Immortal/retile#24]
The capstone of +1 civic is really strong and the rest of it is kinda meh. Info supporting claim to come v. soon.

**Comment by gebnar** [comment]
Thanks for pointing out this issue. We'll be overhauling the tradition trees at some point, so for now we're just setting up band-aid fixes while we get the more core features working well. I'll keep this issue open as it's a valuable discussion to have, and will help us going forward.

**Comment by weirfish** [comment]
So I've been evaluating the trees, and I think the issue is primarily that Diplomacy is so strongly weighted towards its capstone that the rest of it is pretty crap. However, given it's diplomacy, it's probably not worth a rebalance until after the diplomacy DLC. Once that's landed, it's probably worth having a good look at it, but if you're overhauling it entirely then, it's probably not ultimately an issue. 

I won't close this in case you want it to stay open, but it's probably the last I'll say on it until after the overhaul.

**Comment by gebnar** [comment]
Yeah, I'll leave it open until we do something about it. That way others who have similar concerns can see what our plans are.
